### PR TITLE
Add ngscheckmate

### PR DESCRIPTION
**Pull request by cruk-mi**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/ngscheckmate/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd $PREFIX
+mv $SRC_DIR $PREFIX/NGSCheckMate-1.0.0
+
+mkdir -p $PREFIX/bin
+cat << EOF > $PREFIX/bin/ncm.py
+#!/usr/bin/env bash
+
+python /usr/local/NGSCheckMate-1.0.0/ncm.py "\$@"
```
Comment (rpetit3) [inline recipes/ngscheckmate/build.sh:10]: We'll need to think about this, because on the conda installs `ncm.py` will not go to `/usr/local/`

Comment (cruk-mi) [reply recipes/ngscheckmate/build.sh:10]: Thank you for this feedback. The main issue I'm having currently is for some of my recent commits the DevOps tests have gone beyond the walltime of 6 hours and have subsequently failed (e.g. see commit 435daae). I believe it's linked to the requirements I included in the meta.yaml file; the problem goes away if I don't include R in the requirements. Do you know why this is happening?

Comment (rpetit3) [reply recipes/ngscheckmate/build.sh:10]: Can the Python and Samtools version requirements be loosened a bit?

Comment (cruk-mi) [reply recipes/ngscheckmate/build.sh:10]: Hi, thanks for looking at this. I made the version requirements as loose as possible (python needs to be 2.7*), still looks to be having issues. The bit it seems to get stuck on is while collecting package metadata (repodata.json).

Comment (rpetit3) [reply recipes/ngscheckmate/build.sh:10]: I played around, and yeah it takes forever to solve the environment. I used Mamba and it solved it pretty quickly. I've pinned the versions that mamba picked, then reran with conda and it solved it fine.

Are the versions ok? 
```
python 2.7.18
R 4.1
samtools 1.14
```

Comment (cruk-mi) [reply recipes/ngscheckmate/build.sh:10]: I've given it a test and those versions are good. Thanks a lot, I'm going to have a think about how to handle ncm.py

**Comment by cruk-mi** [comment]
@BiocondaBot please add label

**Comment by rpetit3** [comment]
@cruk-mi think your all set now?

**Review by rpetit3 — APPROVED** [review]

**Comment by cruk-mi** [comment]
> @cruk-mi think your all set now?
@rpetit3 Yep! Bit of a learning curve this one. Thanks for the help Robert.

**Comment by rpetit3** [comment]
Awesome! Glad you were able to get it up and going! Thank you for adding this tool

**Pull request merged**